### PR TITLE
Make NETHACKOPTIONS a tuple to protect from modification.

### PR DESCRIPTION
**Pull request by yobibyte**

### CODE REVIEW

File: nle/nethack/nethack.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def __init__(
         shutil.copyfile(DLPATH, dlpath)
 
         if options is None:
-            options = NETHACKOPTIONS
+            options = list(NETHACKOPTIONS)
```
Comment (heiner) [inline nle/nethack/nethack.py:118]: Is this `list` call necessary given the next line?

**Review by heiner — APPROVED** [review]
I like it. Let's wait for the tests to pass before submitting though.

File: nle/nethack/nethack.py (hunk 1 above)
Comment (yobibyte) [reply nle/nethack/nethack.py]: You're right!

**Review by heiner — APPROVED** [review]
LGTM.

**Pull request merged**